src/lib/plugins/BrowserPlugin/BrowserPlugin.ts

```diff
@@ -44,6 +44,9 @@ export function createBrowserPlugin({ history }: BrowserPluginProps): PluginCons
 
     /** Sync the history state when a page transition occurs */
     handlePageTransition: History.LocationListener = (location, action) => {
+      // XXX: This will reflect the outgoing location when the page loads.
```

Comment:
What does XXX: mean?

Reply:
https://softwareengineering.stackexchange.com/questions/65467/what-does-xxx-mean-in-a-comment